source/History.js

```diff
@@ -353,6 +366,9 @@
 
 			switch (state) {
 			case 'empty':
+				if (history.state.currentObjId == 'historyMaster') {
```

Comment:
Hi @aarontam , thank you for your point. 
For this reason I added an {{_initHistoryState}} method.
But I observed that there are some cases which above method was not called before I pressed back key.
For safety, I add one more condition. Thank you.